app/models/miq_expression.rb

```diff
@@ -1195,7 +1195,7 @@ def self.operands2rubyvalue(operator, ops, context_type)
         fld = ref ? "<value ref=#{ref}, type=#{col_type}>#{val}</value>" : "<value type=#{col_type}>#{val}</value>"
         ret.push(fld)
         if ["like", "not like", "starts with", "ends with", "includes", "regular expression matches", "regular expression does not match", "ruby"].include?(operator)
-          ret.push(ops["value"])
+          ret.push(ops["value"].to_s.downcase)
```

Comment:
I'm wondering if it could be better to use Regexp's ignore case option for matching in Ruby?